NuRadioReco/utilities/fft.py

```diff
@@ -47,4 +56,9 @@ def freq2time(spectrum, sampling_rate, n=None):
     n: int
         the number of sample in the time domain (relevant if time trace has an odd number of samples)
     """
-    return np.fft.irfft(spectrum, axis=-1, n=n) * sampling_rate / 2 ** 0.5
+    spectrum_copy = spectrum.copy()
+    # All frequencies components apart from the 0-frequency and (for even-length traces) the maximum frequency
+    # are modified with respect to the numpy convention in NuRadioMC. This is accounted for in the next line
+    n_is_even = (n is None) or ((n + 1) % 2)
+    spectrum_copy[len(spectrum)%2:len(spectrum) - n_is_even] /= np.sqrt(2)
```

Comment:
I am a bit confused by the `len(spectrum) % 2`. When the number of samples is even, i.e., `len(spectrum) % 2 = 0` there is no 0-frequency bin? 

Also please add spaces around `%`

Reply:
Good catch, thanks.